doc/tutorials/model.rst

```diff
@@ -119,13 +119,16 @@ Let the following be the objective function (remember it always needs to contain
 
 .. math::
 
-  \text{obj} = \sum_{i=1}^n l(y_i, \hat{y}_i^{(t)}) + \sum_{i=1}^t\omega(f_i)
+  \text{obj} = \sum_{i=1}^n l(y_i, \hat{y}_i^{(t)}) + \sum_{k=1}^t\omega(f_k)
+
+in which :math:`t` is the number of trees in our ensemble.
```

Comment:
Added this because `t` has not yet been introduced. Another solution would be to let the formula sum from `k` to `K` (as in previous section) and delay the introduction of `t` until the next section.